lib/variable/reduction.cpp

```diff
@@ -22,148 +22,104 @@ using namespace scipp::core;
 namespace scipp::variable {
 
 namespace {
-
-Variable make_accumulant(const Variable &var, const Dim dim,
+/// Note that masking is not supported here since it would make creation
+/// of a sensible starting value difficult. So masks must be applied
+/// before calling reduce_idempotent.
```

Comment:
Renamed?

Reply:
Good catch! Also, now this comment is definitely not true anymore.